documentation/asciidoc/topics/proc_defining_environment_var.adoc

```diff
@@ -0,0 +1,51 @@
+[id='defining-environment-variables_{context}']
+= Defining environment variables in the {ispn_operator} subscription
+
+[role="_abstract"]
+Define environment variables in your {ispn_operator} subscription that automatically propagate to each {brandname} po and service managed by the corresponding Operator.
```

Comment:
`s/po/pod`

The environment variables will not be set on a Service as there's no environment associated with this.